app/react/App.tsx

```diff
@@ -19,11 +21,13 @@ const recoilGlobalState = ({ set }: MutableSnapshot) => {
 
 const App = () => (
   <Provider store={store as any}>
-    <CustomProvider>
-      <RecoilRoot initializeState={recoilGlobalState}>
-        <RouterProvider router={router} fallbackElement={null} />
-      </RecoilRoot>
-    </CustomProvider>
+    <DndProvider backend={HTML5Backend}>
```

Comment:
Is it neccesary to wrapp the entire app? Could we not just use the provider when a route renders a table with dnd?

Reply:
I've moved the provider locally